Formula/stackql.rb

```diff
@@ -0,0 +1,29 @@
+class Stackql < Formula
+  desc "SQL interface for arbitrary resources with full CRUD support"
+  homepage "https://stackql.io/"
+  url "https://github.com/stackql/stackql.git",
+    tag:      "v0.3.265",
+    revision: "ce58a3b9cd00022da42cb8144922613e9491f2f3"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "1"
+    ldflags = [
+      "-s",
+      "-w",
+      "-X github.com/stackql/stackql/internal/stackql/cmd.BuildMajorVersion=0",
+      "-X github.com/stackql/stackql/internal/stackql/cmd.BuildMinorVersion=3",
+      "-X github.com/stackql/stackql/internal/stackql/cmd.BuildPatchVersion=265",
+      "-X github.com/stackql/stackql/internal/stackql/cmd.BuildCommitSHA=#{Utils.git_head}",
+      "-X github.com/stackql/stackql/internal/stackql/cmd.BuildShortCommitSHA=#{Utils.git_short_head}",
+      "-X \"stackql/internal/stackql/planbuilder.PlanCacheEnabled=true\"",
```

Comment:
```suggestion
      "-X stackql/internal/stackql/planbuilder.PlanCacheEnabled=true",
```
The quotes don't seem necessary.

Reply:
@carlocab done cheers